client/signup/steps/free-plans-domain-discount-launch/index.jsx

```diff
@@ -0,0 +1,192 @@
+		</div>
+	);
+}
+
+function RecommendedDomains( props ) {
+	const {
+		signupDependencies: { siteSlug },
+	} = props;
+	const translate = useTranslate();
+	const [ selectedDomain, setSelectedDomain ] = useState( null );
+	const selectDomain = ( event ) => setSelectedDomain( event.target.value );
+	const secureYourBrand = useSelector( ( state ) => getSecureYourBrand( state ) );
+	const isLoading = useSelector( ( state ) => isRequestingSecureYourBrand( state ) );
+	const domain = getDomainName( siteSlug );
+	const productData = secureYourBrand.product_data;
+	const selectedProduct = productData?.filter(
+		( product ) => product.domain === selectedDomain
+	)[ 0 ];
+
+	useEffect( () => {
+		if ( productData && ! selectedDomain ) {
+			setSelectedDomain( productData[ 0 ].domain );
+		}
+	}, [ productData, selectedDomain, setSelectedDomain ] );
+
+	const handleUpgradeButtonClick = () => {
+		const { additionalStepData, stepSectionName, stepName, submitSignupStep, goToNextStep } = props;
+		if ( ! selectedProduct ) {
+			return;
+		}
+
+		const selectedDomainUpsellItem = domainRegistration( {
+			domain: selectedProduct.domain,
+			productSlug: selectedProduct.product_slug,
+		} );
+		const step = {
+			stepName,
+			stepSectionName,
+			selectedDomainUpsellItem,
+			...additionalStepData,
+		};
+
+		submitSignupStep( step, { selectedDomainUpsellItem } );
+		goToNextStep();
+	};
+
+	const handleSkipButtonClick = () => {
+		const { additionalStepData, stepSectionName, stepName, submitSignupStep, goToNextStep } = props;
+
+		const step = {
+			stepName,
+			stepSectionName,
+			selectedDomainUpsellItem: null,
+			...additionalStepData,
+		};
+
+		submitSignupStep( step, { selectedDomainUpsellItem: null } );
+		goToNextStep();
+	};
+
+	return (
+		<div className="free-plans-domain-discount-launch">
+			{ ! productData && <QuerySecureYourBrand domain={ domain } /> }
+			<Card style={ { maxWidth: '615px' } } className="free-plans-domain-discount-launch__card">
+				{ isLoading ? (
+					[ ...Array( 3 ) ].map( ( i ) => (
+						<div key={ i } className="free-plans-domain-discount-launch__placeholder">
+							<div className="free-plans-domain-discount-launch__placeholder-animation"></div>
+						</div>
+					) )
+				) : (
+					<FormFieldset>
+						{ productData?.map( ( suggestion, index ) => (
+							<FormLabel
+								className="free-plans-domain-discount-launch__domain-radio"
+								key={ suggestion.domain }
+							>
+								<FormRadio
+									checked={ selectedDomain === suggestion.domain }
+									onChange={ selectDomain }
+									label={ FormattedSuggestion( translate, suggestion, index === 0 ) }
+									value={ suggestion.domain }
+								/>
+							</FormLabel>
+						) ) }
+					</FormFieldset>
+				) }
+				<div className="free-plans-domain-discount-launch__buttons">
+					<Button busy={ isLoading } primary onClick={ handleUpgradeButtonClick }>
+						{ isLoading ? '' : translate( 'Use %s', { args: [ selectedDomain ] } ) }
```

Comment:
```suggestion
						{ isLoading ? translate( 'Loading' ) : translate( 'Use %s', { args: [ selectedDomain ] } ) }
```
Would it make sense to add the text "Loading" here?
During the loading state, it currently looks like this:
![Kapture 2020-12-08 at 12 05 22](https://user-images.githubusercontent.com/5436027/101448885-d1335780-394d-11eb-89c9-222437f4411c.gif)

Reply:
Yep. Adding "Loading" makes sense. 👍